blocks/api/raw-handling/test/integration/anchor-image-in.html

```diff
@@ -0,0 +1 @@
+<a href="#"><img src="http://http://example.com/" alt="tapestry" width="400" height="800"></a>
```

Comment:
May it be worth making a subfolder for the image block?

Reply:
Yes I will add it :)